package.json

```diff
@@ -53,33 +53,33 @@
     "test:types": "vue-tsc --noEmit && cd playground && vue-tsc --noEmit"
   },
   "dependencies": {
-    "@nuxt/kit": "^3.11.2",
+    "@nuxt/kit": "^3.12.3",
     "defu": "^6.1.2",
     "pathe": "^1.1.1"
   },
   "devDependencies": {
     "@nuxt/eslint-config": "0.3.13",
     "@nuxt/module-builder": "0.8.1",
-    "@nuxt/schema": "3.12.2",
-    "@nuxt/scripts": "0.5.1",
+    "@nuxt/schema": "3.12.3",
+    "@nuxt/scripts": "0.6.2",
     "@nuxt/test-utils": "3.13.1",
-    "@types/cloudflare-turnstile": "0.1.5",
-    "@types/node": "20.14.9",
+    "@types/cloudflare-turnstile": "0.2.0",
+    "@types/node": "20.14.10",
     "@vitest/coverage-v8": "1.6.0",
     "bumpp": "9.4.1",
     "conventional-changelog-conventionalcommits": "8.0.0",
-    "eslint": "9.5.0",
+    "eslint": "9.6.0",
     "expect-type": "0.19.0",
     "get-port-please": "3.1.2",
     "lint-staged": "15.2.7",
-    "nuxt": "3.12.2",
-    "playwright": "1.45.0",
-    "typescript": "5.5.2",
+    "nuxt": "3.12.3",
+    "playwright": "1.45.1",
+    "typescript": "5.5.3",
     "vitest": "1.6.0",
-    "vue-tsc": "2.0.24"
+    "vue-tsc": "2.0.26"
   },
   "peerDependencies": {
-    "@nuxt/scripts": ">=0.5.0 || <0.6.0"
+    "@nuxt/scripts": ">=0.6.0 || <0.7.0"
```

Comment:
Won't this match every possible version?

Reply:
Whoops, you're correct. I think `^0.6.0` is what I mean which should be an alias for `>=0.6.0 <0.7.0`. It should stay below 0.7.0 as that would likely contain breaking changes. Do you agree?

Reply:
Yes 👍

Reply:
changed :raised_hands: